### PR TITLE
Command prestige gear

### PR DESCRIPTION
**Pull request by Sha-Seng**
:cl:
- add: Captain now has a whole 50-hour prestige set (functionally identical!)
- tweak: Adjusted the Captain's sword sheath to stop covering our pretty uniform belts with a plain brown line
- add: All Command spawns with a command survival box (CE gets an extended one, HoS gets a special gas mask!)
- add: All Command now have 50-hour prestige bag and headset options
- add: A cool box of cigarettes hehe

### CODE REVIEW

**Comment by Darkmajia** [comment]
the VeteranCommand loadout group only needs to be defined once, putting it in every group is throwing errors. probably just leave it in captain and get rid of the others? maybe theres a better spot for it but idk

**Comment by iaada** [comment]
I'm guessing that linter is getting upset that you've copy pasted a few prototypes. From a glance probably the 50 hour command timer and command bags. What you need to do is create a new yml file where all of that is defined a single time. You don't need an individual copy in each of the command role files, because all of those files are getting mashed together and read before `loadout-groups.yml` starts looking for the loadout prototypes.

For example on my similar PR, you can see that all of my loadouts are [defined a single time in one file](https://github.com/impstation/imp-station-14/pull/316/files#diff-255cf1b40ca69cab7ed1671afe7935e0824c8c7fc1741a3d71de751a1632e033). Those singular loadouts then get re-used multiple times in different `loadoutGroup`s.

File: Resources/Prototypes/Loadouts/Jobs/Engineering/chief_engineer.yml [hunk 1]
```diff
@@ -8,6 +8,16 @@
       role: JobChiefEngineer
       time: 72000 #20 hrs
 
+# For the veteran command cosmetic set
+- type: loadoutEffectGroup
+  id: VeteranCommand
+  effects:
+  - !type:JobRequirementLoadoutEffect
```
Comment (TGRCdev) [inline Resources/Prototypes/Loadouts/Jobs/Engineering/chief_engineer.yml:15]: You've copied this object across multiple files with the same ID, which is causing YML errors. Please define it once and delete every other instance of it.

**Review by TGRCdev — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Loadouts/Jobs/Engineering/chief_engineer.yml (hunk 1 above)
Comment (TGRCdev) [reply Resources/Prototypes/Loadouts/Jobs/Engineering/chief_engineer.yml:15]: Ah, my bad! I just jumped straight into the "Files Changed" tab 😅

**Pull request merged**